docs/business-apps/get-started/set-up-sharepoint-site-lists-libraries.md

```diff
@@ -1,20 +1,25 @@
 ---
 title: Set up your SharePoint site with lists and libraries
 description: Set up your SharePoint site with lists and libraries
-ms.date: 12/12/2018
+ms.date: 3/11/2020
 ms.prod: sharepoint
 localization_priority: Priority
 ---
 
 # Set up your SharePoint site with lists and libraries
-
 To successfully complete the *Getting Started* tutorials, set up your SharePoint site with the following items.
 
-## Projects list
+## Microsoft 365 tenant
+To use Power Automate and build flows, you need an Office 365 tenant.
+
+If you already have an Office 365 tenant, see section **Create Projects list** below.
+
+If you don't have one, you can get an Office 365 developer subscription when you join the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program). See the [Office 365 Developer Program documentation](https://docs.microsoft.com/office/developer-program/office-365-developer-program) for step-by-step instructions about how to join the Office 365 Developer Program and sign up and configure your subscription.
```

Comment:
Microsoft 365 developer subscription v Office 365 developer subscription.

Flip-flop the following: For step-by-step instructions about how to join the Microsoft 365 Developer Program and sign up and configure your subscription, see the [Office 365 Developer Program documentation].